test_game.rb

```diff
@@ -0,0 +1,122 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+# require_relative 'game'
+
+class TestGame < Minitest::Test
+  def test_winner_not_winning_row
+    g = Game.new
```

Comment:
Operacje które powinny być wykonywane przed każdym testem można zawrzeć na przykład w metodzie `minitest#setup`, tak żeby nie powtarzać tego samego kodu. (https://chriskottom.com/blog/2014/10/4-fantastic-ways-to-set-up-state-in-minitest/).